src/test/test/java/seedu/address/testutil/AddressBookBuilder.java

```diff
@@ -1,34 +1,34 @@
-package seedu.address.testutil;
+package seedu.realtime.testutil;
 
-import seedu.address.model.AddressBook;
-import seedu.address.model.person.Person;
+import seedu.realtime.model.realTime;
+import seedu.realtime.model.person.Person;
 
 /**
- * A utility class to help with building Addressbook objects.
+ * A utility class to help with building realTime objects.
  * Example usage: <br>
- *     {@code AddressBook ab = new AddressBookBuilder().withPerson("John", "Doe").build();}
+ *     {@code realTime ab = new RealTimeBuilder().withPerson("John", "Doe").build();}
  */
-public class AddressBookBuilder {
+public class RealTimeBuilder {
 
-    private AddressBook addressBook;
+    private realTime realTime;
```

Comment:
Somewhere in the refactoring, it has refactored `AddressBook` to `realTime`, just need to now refactor the realTime.java to RealTime.java and it should change this as well when refactoring